apps/etterlatte-utbetaling/src/main/kotlin/no/nav/etterlatte/utbetaling/simulering/SimuleringOsService.kt

```diff
@@ -68,49 +66,57 @@ class SimuleringOsService(
         }
 
     private fun tilOppdrag(
-        vedtak: Vedtak,
+        vedtak: VedtakDto,
         brukerTokenInfo: BrukerTokenInfo,
     ): Oppdrag {
         return Oppdrag().apply {
-            fagsystemId = vedtak.sakId.toString()
-            oppdragGjelderId = vedtak.soeker.value
+            fagsystemId = vedtak.sak.id.toString()
+            oppdragGjelderId = vedtak.sak.ident
             saksbehId = brukerTokenInfo.ident()
 
             utbetFrekvens = "MND"
             kodeEndring = vedtak.type.toKodeEndring()
-            kodeFagomraade = vedtak.sakType.toKodeFagomrade()
+            kodeFagomraade = vedtak.sak.sakType.toKodeFagomrade()
 
-            if (vedtak.innhold is VedtakInnhold.Behandling) {
-                datoOppdragGjelderFom = vedtak.innhold.virkningstidspunkt.atDay(1).toOppdragDate()
+            val innhold = vedtak.innhold
+            if (innhold is VedtakInnholdDto.VedtakBehandlingDto) {
+                datoOppdragGjelderFom = innhold.virkningstidspunkt.atDay(1).toOppdragDate()
                 oppdragslinje.addAll(
-                    vedtak.innhold.utbetalingsperioder.map {
+                    innhold.utbetalingsperioder.map {
                         tilOppdragsLinje(
                             it,
                             vedtak,
                             brukerTokenInfo,
                         )
                     },
                 )
+                enhet.add(
```

Comment:
`OppdragDefaults` har noe av denne infoen hvis du vil gjenbruke det, men det er kanskje på et annet namespace

Reply:
Tenkte å ta all oppdatering/endring av tjenesten separat :) for ikke å ha altfor mye i en